params.js

```diff
@@ -8,10 +8,16 @@ const seleniumPath = require('selenium-server-standalone-jar').path;
 
 const root = dirname(findUp.sync('package.json'))
 const picturebookPath = 'node_modules/picturebook/'
-const explorer = cosmiconfig(name, {
+let explorer = cosmiconfig(name, {
   cache: false,
   sync: true
-})
+}).load(__dirname)
+
+if (!explorer) {
```

Comment:
Does picturebook work without a config? If so, this works, otherwise maybe we should just throw?

Reply:
It depends on the use case, for simple cases it should be 0 config but for the both projects where we use it we need it. I'm fine either way.

This error occurred when I was installing `picturebook` at the top level and I had the config within a workspace page, then it failed to find the config and returned `null` which resulted in picturebook crashing. I don't mind not logging the warning either if we feel like it's overkill